### ISSUE TITLE
Example

### ISSUE THREAD

**Issue by LordRias** [lennybakkalian/GameEngine#1]
Du solltest die gameengine von deinem Spiel trennen

**Comment by lennybakkalian** [comment]
Stimmt, hast recht! xD Aber Nur die LoadingState Class und die champions/ability klassen beinhalten beispielcode zum testen. Die engine ist noch nicht fertig.
